packages/jest-cli/src/cli/index.js

```diff
@@ -103,12 +104,39 @@ export const runCLI = async (
   const {openHandles} = results;
 
   if (openHandles && openHandles.length) {
-    const openHandlesString = pluralize('open handle', openHandles.length, 's');
+    const formatted = formatHandleErrors(openHandles, configs[0]);
+
+    const stacks = new Set();
+
+    const uniqueErrors = formatted.filter(handle => {
```

Comment:
this whole function is really hairy. ideas?

Reply:
IDK. `TIMEWRAP` is a libuv thing, that's what I got from googling it. I don't think it really matters, they point to the same thing

Reply:
Or maybe we can just ditch the filtering?

Reply:
@cpojer wanted to filter out the dupe

Reply:
Can you just add a comment about the dupes? Should be good enough

Reply:
done